errors/app-dir-dynamic-href.mdx

```diff
@@ -1,4 +1,8 @@
-# Dynamic `href` is not supported in the `/app` directory
+---
+title: 'Dynamic `href` is not supported in the `/app` directory'
+---
+
+## `app-dir-dynamic-href`
```

Comment:
It'd be great to render the `title` from the frontmatter, because the slug here isn't meaningfully describing the error.